index.js

```diff
@@ -9,13 +9,16 @@ exports.buildInspectorUrl = buildInspectorUrl;
  * @param {number} inspectorPort as configured via --web-port
  * @param {number} debugPort as configured via --debug in the debugged app
  */
-function buildInspectorUrl(inspectorHost, inspectorPort, debugPort, fileToShow) {
+function buildInspectorUrl(inspectorHost, inspectorPort, debugHost, debugPort, fileToShow) {
```

Comment:
This is a public API, we need to preserve backwards compatibility. Please add a bit of argument-manipulation to support the old flavour `(inspectorHost, inspectorPort, debugPort, fileToShow)` too.